src/dashboard/testUtils/index.js

```diff
@@ -3,14 +3,21 @@ import { ApolloProvider } from 'react-apollo-hooks';
 import { render } from '@testing-library/react';
 
 import { createApolloMockClient } from './apolloMockClient';
+import defaultTheme from '../defaultTheme';
+import { ThemeProvider } from 'emotion-theming';
+import AppContext from '../appContext';
 
-export const renderWithContext = (component, { mocks }) => {
+export const renderWithContext = (component, { mocks } = {}, context = {}) => {
```

Comment:
I think it would be nicer to keep the number of parameters down here since you may not use them all each time but would have to provide null or something if you wanted to skip one.
```suggestion
export const renderWithContext = (component, { mocks, context = {} } = {}) => {
```

Also `context` seems a little general. `appContext` maybe? Idk. Not sure how many values we plan on sticking in here...